### ISSUE TITLE
“command 'vscodeai.openGallery' not found”

### ISSUE THREAD

**Issue by azeltov** [Microsoft/vscode-tools-for-ai#25]
Installed vstools for ai and now when i try to open a sample azure ml , i get this error: “command 'vscodeai.openGallery' not found”

**Comment by qfyin** [comment]
Thanks for using vscode tools for ai.
I can't repro this issue.
Please try the latest version 0.1.8

**Comment by mikhey** [comment]
Just installed 0.4.5, never saw a message about installing Azure ML, I get error pop-ups about commands not being found.

> command 'vscodeai.azureml.workspace.add' not found

> command 'vscodeai.view.azureml.refresh' not found

Setup:
OS: Mac OSX High Sierra
VS Code Version: 1.32.3
Azure ML VS Code Extension: 0.4.5

**Comment by greazer** [comment]
These symptoms sound like the extension is either not installed correctly for some reason, or isn't getting through VSCode's extension registration step. Please open your settings and find "ai logger". Change AI > Logger > Level: Channel to "debug". Then reload VSCode and let me know what shows up in your output window when you attempt to open and/or use any of the extension.

**Comment by greazer** [comment]
Reopening for the time being, so as to not lose track of this new issue.

**Comment by mikhey** [comment]
I enabled the logs, but I didn't get much:

```
debug: Azure ML for VSCode 0.4.5
debug: Platform: darwin
```

Just to test, I installed 0.4.2, I had some success, it showed a box notifying me that I'm not signed in:
```
Not signed in, sign in first.
```
<img width="461" alt="Screen Shot 2019-03-27 at 7 23 15 PM" src="https://user-images.githubusercontent.com/449275/55118913-53baab80-50c6-11e9-996a-48afaa56bcdc.png">


However, after the extension auto updates to 0.4.5 and I reload VS Code, I get the same error message pop-up:
```
command 'vscodeai.azureml.workspace.add' not found
```
<img width="463" alt="Screen Shot 2019-03-27 at 7 25 05 PM" src="https://user-images.githubusercontent.com/449275/55118873-31289280-50c6-11e9-886e-d66caf09caf7.png">

Maybe my install for 0.4.5 was interrupted/corrupted.  What's the best option to completely remove the extension and try again?

**Comment by greazer** [comment]
Yes, lets try that first. To uninstall completely, first uninstall using the regular VSCode UI. But then on mac, go to the ~/.vscode/extensions folder and delete the folder named ms-toolsai.vscode-ai-0.4.5. Also, delete the whole ~/.azureml directory. Then reinstall.

**Comment by haven-jeon** [comment]
I have same issue. Could not resolve  @greazer 's method.

**Comment by nrobert** [comment]
Same problem here:

OS: Mac OS Mojave (10.14.4)
VS Code Version: 1.33.1
Azure ML VS Code Extension: 0.4.5

Cannot list Machine Learning items, got "There is no data provider registered that can provide view data." in the panel 
When hitting the refresh sign, a pop up shows "command 'vscodeai.view.azureml.refresh' not found"

**Comment by greazer** [comment]
Hmmmm... It sounds like the extension is not loading or is crashing for some reason. @nrobert , @haven-jeon  after you delete the ~/.vscode/extensions/ms-toolsai.vscode-ai-0.4.5 folder and the ~/.azureml folder, and you reinstall the extension, reload VSCode and switch to the Azure tab, do you ever get a popup message at the bottom right stating that the Azure ML extension needs to install runtime dependencies?

**Comment by nrobert** [comment]
I have not seen this popup message no

**Comment by greazer** [comment]
We're seeing a similar situation here in our upcoming release. After starting VSCode and before activating the Azure ML view, make sure you open a folder. This may not be the problem you're running into, but the symptoms we see due to this bug are similar.

**Comment by nrobert** [comment]
Yes, the extension installed dependencies when opening a folder first:

> Please wait a few minutes while we install Azure ML extension runtime dependencies

**Comment by greazer** [comment]
Ok. We added a fix for the original problem in version 0.5.0 as well.
